lib/opensusebasetest.pm

```diff
@@ -639,6 +639,12 @@ sub post_fail_hook {
     # see any interesting console logs.
     send_key 'esc';
     save_screenshot;
+
+    # Find out in post-fail-hook if system is I/O-busy, poo#35877
+    select_console("root-console");
```

Comment:
@rwx788 it works for ipmi as well:
http://f40.suse.de/tests/48#step/logs_from_installation_system/25

Reply:
Please investigate why log-console does not work. We should stick to our convention to use the log console for post_fail_hook as the root console might be blocked by a command.

@rwx788 I seriously hope the root-console still works on all backends. We already accepted quite some changes in the wrong direction and as you know people complained about how the tests look messy e.g.because of "if backend Foo..." code. Hence I look out to just make the plain simple console selections work

Reply:
I'm totally with you, but we have e.g. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/system_prepare.pm#L24 for a reason. And I'm totally with you, we need an abstraction in `select_console` or one of it's callbacks, so test writers do not care what it will be ssh or tty.

Reply:
I remove select_console 'root-console' and put my code before checking output error which has select_console 'log-console'. 
It works now:
http://f40.suse.de/tests/55#step/logs_from_installation_system/20